Sage/carbon
# feat(simple-color-picker, advanced-color-picker): describe components using design tokens, update tests - FE-4734

### Proposed behaviour

Describes `Simple Color Picker` and `Advanced Color Picker` components using design tokens.
Changes the thickness for box-shadows : `var(--borderWidth200)`.
Changes the thickness for outline for the validation type `caution` and `info`: `var(--borderWidth200)`.
In `Advanced Color Picker` changes the color of the border to: `var(--colorsUtilityYin090)`.

### Current behaviour

`Simple Color Picker` and `Advanced Color Picker` use the theme styled-component property or color values that are hardcoded.
Current the thickness for box-shadows: `3px`.
Current the thickness for outline for the validation type `caution` and `info`: `1px`.
Current color of border in `Advanced Color Picker` `#516562`.

### Checklist




#### QA


### Additional context

Simple Color Picker: 
![Zrzut ekranu 2022-02-18 o 13 00 29](https://user-images.githubusercontent.com/44143630/154679440-123774ad-d328-4a22-9dad-4cb90129e446.png)
![Zrzut ekranu 2022-02-18 o 13 01 06](https://user-images.githubusercontent.com/44143630/154679448-caeb718e-9cc0-4410-babe-4d6781596e76.png)

Advanced Color Picker: 
![Zrzut ekranu 2022-02-18 o 13 01 27](https://user-images.githubusercontent.com/44143630/154679546-f640db58-3922-4865-8f9b-91817307db43.png)

### Testing instructions

The correct appearance can be checked in the components: Simple Color Picker, Advanced Color Picker.
Use devtools to check if components use css variables.
